test/src/test/java/hudson/model/FreestyleJobPublisherTest.java

```diff
@@ -0,0 +1,106 @@
+package hudson.model;
```

Comment:
Why do you really need it? Default license is MIT and i marked as `@author` for test class?